lib/wasix/src/state/env.rs

```diff
@@ -1125,18 +1125,17 @@ impl WasiEnv {
                         }
                     }
                     WasiFsRoot::Backing(fs) => {
+                        // FIXME: we're counting on the fs being a mem_fs here. Otherwise, memory
+                        // usage will be very high.
                         let mut f = fs.new_open_options().create(true).write(true).open(path)?;
```

Comment:
Can we get a small regression test for this, just with a temp FS?

Reply:
Would be quite difficult to set up, since we'd need to go through the code path that sets up the WASI FS and then make sure the fs is actually the correct type...